### ISSUE TITLE
Version 1.24 (Crossplay) Patch Update

### ISSUE THREAD

**Issue by crtado** [xynxynxyn/ggst-api-rs#9]
After reading @halvnykterist twitter update regarding the recent patch, I'm assuming there have been changes to the request/response structure of api queries.

What is the current progress on updating to support this? Is there anywhere that y'all need help? I'd be happy to spend some time on it.

**Comment by crtado** [comment]
Also correct me if I'm wrong, but are the following lines the affected structures that have become outdated after the patch? Just clarifying the potential issue for my own understanding.

https://github.com/xynxynxyn/ggst-api-rs/blob/568451ea2ce92160cc9c4ef4a0a98f82acb925d0/src/requests.rs#L356-L428

**Comment by xynxynxyn** [comment]
Yes, that was the previous structure for getting replays. Since totsugeki is also no longer working properly anymore, I'm having trouble actually seeing how the new requests and responses are structured.

If you'd like to help for now, the first step would be to get a functioning proxy (like totsugeki) for the game again. I got no time or the skills for this at the moment, but if you make any progress I'd take a look at it.

**Comment by OOPMan** [comment]
@xynxynxyn I guess something generic like MITMProxy doesn't work?

**Comment by xynxynxyn** [comment]
@OOPMan I don't know for sure, but you might want to check this issue https://github.com/optix2000/totsugeki/issues/86
Looks like they figured it out a while ago and it works on ratingupdate, though it is not implemented in this project yet.
